modules/customizing-templates/proc-creating-templates.adoc

```diff
@@ -4,48 +4,46 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="proc-creating-templates_{context}"]
-= Creating a Software Template by using the Template Editor
+= Creating a Software Template by using the Template Editor Templates
 
-You can create a Software Template by using the Template Editor.
+You can create a Software Template by using the Template Editor templates.
 
 .Procedure
 
-. Access the Template Editor by using one of the following options:
+. In your {product} navigation menu, click *Catalog > Self-service*.
+. Click the overview menu button and select *Manage Templates*.
 +
 image::rhdh/template-editor.png[Template Editor]
-** Open the URL `\https://<rhdh_url>/create/edit` for your {product} instance.
-** Click *Self-service* in the navigation menu of the {product} console, then click the overflow menu button and select *Template editor*.
-. Click *Edit Template Form*.
-. Optional: Modify the YAML definition for the parameters of your template. For more information about these parameters, see <<Creating a Software Template as a YAML file>>.
-. In the *Name* field, enter a unique name for your template.
-. From the *Owner* drop-down menu, choose an owner for the template.
-. Click *Next*.
-. In the *Repository Location* view, enter the following information about the hosted repository that you want to publish the template to:
-.. Select an available *Host* from the drop-down menu.
 +
---
+
 [NOTE]
 ====
-Available hosts are defined in the YAML parameters by the `allowedHosts` field:
-
-.Example YAML
-[source,yaml]
-----
-# ...
-        ui:options:
-          allowedHosts:
-            - github.com
-# ...
-----
-
+* The following options on the *Managed Templates* page do not create a software component in your {product} instance:
+** *Template Form Playground*: Use to create and test the templates.yaml file
+** *Custom Field Explorer*: Use to test custom fields
 ====
---
 
-.. In the *Owner ** field, enter an organization, user or project that the hosted repository belongs to.
-.. In the *Repository ** field, enter the name of the hosted repository.
-.. Click *Review*.
-
-. Review the information for accuracy, then click *Create*.
+. On the *Managed Templates* page, select any of the following options:
+** *Load Template Directory*
+** *Create New Template*
+.. In your local file manager, navigate to the folder where your templates.yaml file is residing and click *Select*.
```

Comment:
These are appearing as a bulleted list instead of a numbered list in the preview.
```suggestion
.. In your local file manager, navigate to the folder where your `templates.yaml` file is residing and click *Select*.
```